### PR TITLE
Updated to Debug Toolbar 1.8

### LINKED ISSUE #3: Not work because of oder of MIDDLEWARE_CLASSES with GZipMiddleware
Not work after installing in my Divio project. Because need placed after  GZipMiddleware in MIDDLEWARE_CLASSES settings.

**Workaround**: copy all middleware setup in MIDDLEWARE_CLASSES in settings.py and move `'debug_toolbar.middleware.DebugToolbarMiddleware'` defined at the beginning just after the middleware  `'django.middleware.gzip.GZipMiddleware'`.

### PR DESCRIPTION
**Pull request by evildmp**
Changes:

* installs Django Debug Toolbar 1.8
* fixes #3, an issue with middleware ordering
* fixes an issue with the `license='BSD'` in setup.py, that was raising an error in local development
* simplified test for actvation in aldryn_config.py
* changed authorship to reflect author of addon package rather than Django Debug Toolbar

### CODE REVIEW

File: aldryn_config.py [hunk 1]
```diff
@@ -15,21 +16,18 @@ def _show_toolbar(self, request):
...
-        env_stage = '' if env('STAGE') is None else str(env('STAGE'))
-        development_mode = True if env_stage in ['test', 'local'] else False
+        if settings["DEBUG"] and data['enable_debug_toolbar']:
+
+            settings["INSTALLED_APPS"].extend(["debug_toolbar"])
+
+            settings["DEBUG_TOOLBAR_CONFIG"] = {"SHOW_TOOLBAR_CALLBACK": self._show_toolbar}
+
+            settings["MIDDLEWARE_CLASSES"].insert(
+                settings["MIDDLEWARE_CLASSES"].index("django.middleware.gzip.GZipMiddleware") + 1,
```
Comment (stefanfoulis) [inline aldryn_config.py]: This will throw an ValueError if ``django.middleware.gzip.GZipMiddleware`` is not in the list.

File: aldryn_config.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 
 class Form(forms.BaseForm):
+
     enable_debug_toolbar = forms.CheckboxField(
```
Comment (stefanfoulis) [inline aldryn_config.py:9]: We should probably remove the field from the form here, and configure from an environment variable instead.

**Review by stefanfoulis — CHANGES_REQUESTED** [review]

**Review by stefanfoulis — APPROVED** [review]
Looks good implementation wise now. I didn't know about the top/bottom of middleswares thing with django-debug-toolbar.
A description in the form that better reflects why this option even exists may be needed.
I'd still like the enable/disable of django-debug-toolbar to be an environment variable, but that we can leave as out-of-scope for this PR.

File: aldryn_config.py [hunk 3]
```diff
@@ -5,31 +5,61 @@
...
+
+    POSITIONS = [
+        ["top", "As close to the top as possible (generally recommended)"],
+        ["end", "As close to the end as possible"],
+
+    ]
+
+    toolbar_middleware_position = forms.SelectField(
+        'DebugToolbarMiddleware position',
+        help_text="Position in the middleware list",
```
Comment (GaretJax) [inline aldryn_config.py:29]: Please consistently use single quotes (except where the string contains a single quote itself). Also below.

File: aldryn_config.py [hunk 4]
```diff
@@ -5,31 +5,61 @@
...
+
+            settings["INSTALLED_APPS"].extend(["debug_toolbar"])
+
+            settings["DEBUG_TOOLBAR_CONFIG"] = {"SHOW_TOOLBAR_CALLBACK": self._show_toolbar}
+
+            middleware = settings["MIDDLEWARE_CLASSES"]
+            debug_toolbar_middleware = "debug_toolbar.middleware.DebugToolbarMiddleware"
+            debug_toolbar_middleware_position = data["toolbar_middleware_position"]
+
+            if "django.middleware.gzip.GZipMiddleware" in middleware and debug_toolbar_middleware_position == "top":
```
Comment (GaretJax) [inline aldryn_config.py:50]: Waaay too long. :)

File: aldryn_config.py [hunk 5]
```diff
@@ -5,31 +5,61 @@
...
+
+                middleware.insert(
+                    middleware.index("django.middleware.gzip.GZipMiddleware") + 1,
+                    debug_toolbar_middleware
+                )
+
```
Comment (GaretJax) [inline aldryn_config.py:56]: Please don't add newlines here (and before the else)

File: aldryn_config.py [hunk 6]
```diff
@@ -5,31 +5,61 @@
...
+    # The generally recommended position for the DebugToolbarMiddleware is immediately after the
+    # GZipMiddleware. See:
+    #
+    # https://django-debug-toolbar.readthedocs.io/en/stable/installation.html#middleware
+    # https://django-debug-toolbar.readthedocs.io/en/stable/tips.html#middleware-isn-t-working-correctly
+
+    POSITIONS = [
+        ["top", "As close to the top as possible (generally recommended)"],
+        ["end", "As close to the end as possible"],
+
```
Comment (GaretJax) [inline aldryn_config.py:24]: Please remove the empty line here.

File: aldryn_config.py [hunk 7]
```diff
@@ -5,31 +5,61 @@
 
 
 class Form(forms.BaseForm):
+
```
Comment (GaretJax) [inline aldryn_config.py:8]: Please remove this empty line.

**Review by GaretJax — APPROVED** [review]
Some minor cosmetic changes; I'll leave it up to you if you want to spend time on it now. ;-)

File: aldryn_django_debug_toolbar/__init__.py [hunk 8]
```diff
@@ -1 +1 @@
-__version__ = "1.5.0"
+__version__ = "1.8.0.4"
```
Comment (jmelett) [inline aldryn_django_debug_toolbar/__init__.py:1]: Version bumps should be done in separate commit

**Pull request merged**